sys/net/gnrc/Makefile.dep

```diff
@@ -57,6 +57,7 @@ endif
 
 ifneq (,$(filter gnrc_dhcpv6_client_simple_pd,$(USEMODULE)))
   USEMODULE += gnrc_dhcpv6_client
+  CFLAGS += -DCONFIG_GNRC_IPV6_NIB_ADV_ROUTER=0
```

Comment:
@leandrolanzieri would this collide with Kconfig?

Reply:
Main question is, if this would be picked up by the Kconfig of `gnrc_ipv6_nib`. I assume it won't and may lead to problems when `gnrc_ipv6_nib` is attempted to be configured using Kconfig.

Reply:
As I understand this line, when `gnrc_dhcpv6_client_simple_pd` module is on, the router advertising should be off. right? Then this would translate into adding a dependency to the option:
```
config GNRC_IPV6_NIB_ADV_ROUTER
    bool "Activate router advertising at interface start-up"
    default y if GNRC_IPV6_NIB_ROUTER && (!GNRC_IPV6_NIB_6LR || GNRC_IPV6_NIB_6LBR)
    depends on !USEMODULE_GNRC_DHCPV6_CLIENT_SIMPLE_PD
```

If it is a default value instead, it should be decided the priority vs. the existing one and added as `default n if USEMODULE_GNRC_DHCPV6_CLIENT_SIMPLE_PD`.

Reply:
so like this? 1215b1f

Reply:
Defaults are evaluated from top to bottom, and the first one that matches the condition is applied. If this new condition does not take precedence to the existing one then it looks good to me.

A note though: as the default is `n`, which is the normal default for `bool`, it may be avoided, but it does not hurt to make it explicit in case another default is included in the future.

Reply:
Ah so it should rather be like this 2a13661